build

```diff
@@ -1 +1 @@
-`dirname $0`/sbt assembly
+/`dirname $0`/sbt assembly
```

Comment:
Why do you add a heading '/' ? It does not work on my machine, while working without it.

Reply:
I think it would be helpful to merge the PR excluding the change to `build`? I tried many times to use scaladiagrams and failed due to dependency problems. The master branch currently runs on a very old version of Java, like Java 5 or something, and that failed for me because I didn't have the correct Java 5 and it's pretty hard to find Java 5 now... This branch just worked for me...

I didn't use `build`. I just ran assembled the jar myself.  

I suspect `build` failed on contra-bit's machine because `directory $0` [behaves differently](http://mywiki.wooledge.org/BashFAQ/028) in different terminals, and ```/`dirname $0`/sbt assembly```  behaves differently on Alexis' terminal than contra-bit's terminal.